clients/fides-js/src/lib/i18n/index.ts

```diff
@@ -56,6 +56,28 @@ interface I18n {
    */
   activate(locale: Locale): void;
 
+  /**
+   * Get the current default locale for this session.
+   *
+   * WARN: LinguiJS does not support getting/setting the default locale, so
+   * use this sparingly!
+   */
+  getDefaultLocale(): Locale;
```

Comment:
Do we have a plan for how we’d do this if / when we switch to LinguiJS?

Reply:
Yeah, we could figure something reasonable out as needed. This wouldn't be too hard.